### DISCUSSION TITLE
Unable to Download Debug build apk

### DISCUSSION THREAD

**Discussion by ghost** [termux/termux-app#2256] [General]
I am going to debug build section on github actions and selecting the latest build flag and can see different architectures available there but on clicking them nothing is getting downloaded.

Should I be building one from the buildyml on linux or I can do the same inside termux on android 64 ?

Also I need to know the process to initiate the build process.

**Comment by agnostic-apollo** [comment]
The latest [`termux-app-arm64-v8a`](https://github.com/termux/termux-app/actions/runs/1193298726) apk is downloading fine. The link that gets attached to releases gets expired at some point, next releases will have apks attached as release assets automatically instead of a link to build action.

For building in termux, check https://github.com/termux/termux-packages/pull/7227#issuecomment-893022283. You will need `openjdk-17` and `gradle` package. For building on linux distro on pc, use android-studio.
